File: laprak2/cpp/main.cpp
#include <iostream>
#include <iomanip>
#include "Human.cpp"

using namespace std;

void printData(list<SivitasAkademik> &sivitasList)
{
    cout << setw(20) << left << "Nama"
         << setw(15) << left << "NIK"
         << setw(20) << left << "Jenis Kelamin"
         << setw(15) << left << "NIM"
         << setw(20) << left << "Prodi"
         << setw(20) << left << "Fakultas"
         << setw(25) << left << "Asal Univ"
         << setw(25) << left << "Email" << endl;

    for (const auto &sivitas : sivitasList)
    {
        cout << setw(20) << left << sivitas.getNama()
             << setw(15) << left << sivitas.getNik()
             << setw(20) << left << sivitas.getKelamin()
             << setw(15) << left << sivitas.getNim()
             << setw(20) << left << sivitas.getProdi()
             << setw(20) << left << sivitas.getFakultas()
             << setw(25) << left << sivitas.getUniv()
             << setw(25) << left << sivitas.getEmail() << endl;
    }
}

int main(int argc, char const *argv[])
{
    string name, fakultas, prodi, kelamin, univ, email;
    long int nik, nim;

    cout << "Masukkan berapa banyak orang mahasiswa : ";

    int n;
    cin >> n;

    list<SivitasAkademik> Slist;

    for (int i = 0; i < n; i++)
    {
        SivitasAkademik temp;

        cout << "Orang ke-" << i + 1 << endl;
        cout << "Name: ";
        cin >> name;
        cout << "NIK: ";
        cin >> nik;
        cout << "Gender: ";
        cin >> kelamin;
        cout << "NIM: ";
        cin >> nim;
        cout << "Major: ";
        cin >> prodi;
        cout << "Faculty: ";
        cin >> fakultas;
        cout << "University: ";
        cin >> univ;
        cout << "Email: ";
        cin >> email;
        cout << endl;

        temp.setNama(name);
        temp.setNik(nik);
        temp.setKelamin(kelamin);
        temp.setNim(nim);
        temp.setProdi(prodi);
        temp.setFakultas(fakultas);
        temp.setUniv(univ);
        temp.setEmail(email);

        Slist.push_back(temp);
    }

    printData(Slist);

    return 0;
}

File: laprak2/php/main.php
<?php
require('Human.php');

$sivitasAkademikList = array(
    new SivitasAkademik("John Doe", 123456789, "Male", 1234567, "Computer Science", "Engineering", "University of XYZ", "john.doe@xyz.edu"),
    new SivitasAkademik("Jane Smith", 987654321, "Female", 7654321, "Electrical Engineering", "Engineering", "University of ABC", "jane.smith@abc.edu")
);

echo "<table border = 2>\n";
echo "<thead>\n";
echo "<tr><th>Full Name</th><th>NIK</th><th>Gender</th><th>NIM</th><th>Program Studi</th><th>Fakultas</th><th>Universitas</th><th>Email</th></tr>\n";
echo "</thead>\n";
echo "<tbody>\n";
foreach ($sivitasAkademikList as $sivitasAkademik) {
    echo "<tr>\n";
    echo "<td>" . $sivitasAkademik->getNama() . "</td>\n";
    echo "<td>" . $sivitasAkademik->getNik() . "</td>\n";
    echo "<td>" . $sivitasAkademik->getKelamin() . "</td>\n";
    echo "<td>" . $sivitasAkademik->getNim() . "</td>\n";
    echo "<td>" . $sivitasAkademik->getProdi() . "</td>\n";
    echo "<td>" . $sivitasAkademik->getFakultas() . "</td>\n";
    echo "<td>" . $sivitasAkademik->getUniv() . "</td>\n";
    echo "<td>" . $sivitasAkademik->getEmail() . "</td>\n";
    echo "</tr>\n";
}
echo "</tbody>\n";
echo "</table>\n";
